docs/README.md

```diff
@@ -1,55 +1,59 @@
-Welcome to DAOstack Arc Docs!
+Welcome to *DAOstack Arc* Docs!
 
-Check out the [Generated docs](generated_docs) for detailed documentation on every contract and an explanation of the source tree.
+*Arc* is the base layer of the DAO stack. It consists of a collection of smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+
+*Arc* is built from the ground up with these principles in mind:
 
-## An Overview of Arc
+3. **Scalable Decentralized Governance** - *Arc* implements game theoretic & economic techniques (such as: monetization of attention, staking on proposal results, and more...) that ensure high resilience and bandwidth of the organization even as the number of the participants gets very large.
+3. **Security** - *Arc* embodies the best security guidelines and is very well tested.
+2. **Interoperability & Compositionality** - *Arc* can integrate with any other ethereum based framework with ease. Organizations can interact with each other and form hierarchies and other complex structures.
+1. **Modularity** - *Arc* is built in a modular way allowing anyone to extend and expand the ecosystem to support new use-cases.
+2. **General purpose** - *Arc* provides a single unified simple model for building any DAO imaginable.
 
-For more: [Read the official DAOstack Whitepaper](https://github.com/daostack/docs/blob/master/DAOstack%20White%20Paper%20V1.0.pdf).
+For more information [read the official DAOstack Whitepaper](https://github.com/daostack/docs/blob/master/DAOstack%20White%20Paper%20V1.0.pdf).
 
-Arc is the lower layer of the DAOStack. It consists of several smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+Check out the [Generated docs](generated_docs) for detailed documentation on every contract and an explanation of the source tree.
```

Comment:
Add a link and explanation to the manual documentation (contracts) also ..

Reply:
The generated docs also include the manual docs as headers. 
I think it's better to ditch the headers thing, and just link from the generated docs from the manual docs. There will be no duplication then. 
What do you think?

Reply:
updated at: https://github.com/daostack/arc/pull/309/commits/ec72cd04e458beaad55e670c756d3a254a6be84c

If this is not wanted, we can undo this commit.